test/unit/ngsiv2/expressions/expressionBasedTransformations-test.js

```diff
@@ -0,0 +1,774 @@
+                        type: 'Object',
+                    },
+                    {
+                        object_id: 'r',
+                        name: 'revisions',
+                        type: 'Array',
+                    },
+                ]
+            },
+            'LightError': {
+                commands: [],
+                type: 'Light',
+                lazy: [],
+                active: [
+                    {
+                        object_id: 'p',
+                        name: 'pressure',
+                        type: 'Integer',
+                        expression: '${@pressure * / 20}'
+                    }
+                ]
+            },
+            'WeatherStation': {
+                commands: [],
+                type: 'WeatherStation',
+                lazy: [],
+                active: [
+                    {
+                        object_id: 'p',
+                        name: 'pressure',
+                        type: 'Integer',
+                        expression: '${@pressure * 20}'
+                    },
+                    {
+                        object_id: 'e',
+                        name: 'consumption',
+                        type: 'Float',
+                        expression: '${@consumption * 20}'
+                    },
+                    {
+                        object_id: 'h',
+                        name: 'humidity',
+                        type: 'Percentage'
+                    },
+                    {
+                        name: 'weather',
+                        type: 'Summary',
+                        expression: 'Humidity ${@humidity / 2} and pressure ${@pressure * 20}'
+                    },
+                    {
+                        object_id: 'a',
+                        name: 'alive',
+                        type: 'Null',
+                        expression: '${@alive *  20}'
+                    },
+                    {
+                        object_id: 'u',
+                        name: 'updated',
+                        type: 'Boolean',
+                        expression: '${@updated *  20}'
+                    },
+                ]
+            },
+            'WeatherStationMultiple': {
+                commands: [],
+                type: 'WeatherStation',
+                lazy: [],
+                active: [
+
+                    {
+                        object_id: 'p',
+                        name: 'pressure',
+                        type: 'Integer',
+                        expression: '${trim(@spaces)}'
+                    },
+                    {
+                        object_id: 'p25',
+                        name: 'pressure25',
+                        type: 'Integer'
+                    },
+                    {
+                        object_id: 'e',
+                        name: 'consumption',
+                        type: 'Float',
+                        expression: '${trim(@spaces)}'
```

Comment:
There isn't any attribute named `spaces`... maybe I'm missing something?

Reply:
Fixed in 4026d75714dd6ef51230bd01c8482c0a89e36923